dev_tools/packaging/verify-published-package.sh

```diff
@@ -44,17 +44,17 @@ if [ -z "${PROJECT_VERSION}" ]; then
 fi
 
 if [ "${PROD_SWITCH}" = "--test" ]; then
-    PYPI_REPOSITORY_FLAG="--index-url=https://test.pypi.org/simple/"
+    PIP_FLAGS="--index-url=https://test.pypi.org/simple/"
     PYPI_REPO_NAME="TEST"
     PYPI_PROJECT_NAME="cirq"
 elif [ "${PROD_SWITCH}" = "--prod" ]; then
-    PYPI_REPOSITORY_FLAG=''
+    PIP_FLAGS=''
     PYPI_REPO_NAME="PROD"
     PYPI_PROJECT_NAME="cirq"
 elif [ "${PROD_SWITCH}" = "--unstable" ]; then
```

Comment:
Optional: Consider renaming the flag to --pre. Alternatively, you can add a second flag to serve as an alias.

(If you choose to do this, update the top comment, too.